go/workflow_test.go

```diff
@@ -0,0 +1,27 @@
+package app
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"go.temporal.io/sdk/testsuite"
+)
+
+func Test_Workflow(t *testing.T) {
```

Comment:
I wonder if we can use the unit tests to show the compensation in practice ie have some unit tests where some operations fail and the compensation needs to happen and put expectations on only some of the compensations happening.

Reply:
yes, fixed that up. Thanks!